lib/protocols.js

```diff
@@ -750,6 +750,7 @@
       return f.dispatch(
         f.isa('application/json', encoder(2, encoderJson, decoderJson()))
         , f.isa('text/plain', encoder(13, encoderString, decoderString()))
+        , f.isa('application/x-protostream', encoder(13, encoderProtobuf, decoderProtobuf()))
```

Comment:
```suggestion
        , f.isa('application/x-protostream', encoder(12, encoderProtobuf, decoderProtobuf()))
```
values for media types are defined [here](Correct value for this is in https://github.com/infinispan/infinispan/blob/10220f7910942e2d2190c3933bd985ac0f6c493e/commons/all/src/main/java/org/infinispan/commons/dataconversion/MediaTypeIds.java#L44)